lib/apipie_bindings/api.rb

```diff
@@ -224,7 +224,7 @@ def http_call(http_method, path, params={}, headers={}, options={})
         rescue => e
           clear_credentials if e.is_a? RestClient::Unauthorized
           log.debug e.message + "\n" +
-            inspect_data(e.respond_to?(:response) ? process_data(e.response) : e)
+            (inspect_data(e.respond_to?(:response) ? process_data(e.response) : e)) rescue ''
```

Comment:
If the issue is in `process_data`, the rescue should go to process_data as well.